code/game/objects/items/weapons/implants/implantchair.dm

```diff
@@ -79,8 +79,8 @@
 		if(!ismob(G.affecting))
 			return
 		var/mob/M = G.affecting
-		if(M.buckled_mob)
-			usr << "[M] will not fit into [src] because they have a slime latched onto their head."
+		if(M.buckled_mobs.len)
+			usr << "[M] will not fit into [src] because they have a creature on them!"

```

Comment:
usr -> user , also it says creature (singular) even if there's more than one mobs buckled.